app/main.py

```diff
@@ -16,4 +16,25 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variable_names = [
+    "lucky_number",
+    "pi",
+    "one_is_a_prime_number",
+    "name",
+    "my_favourite_films",
+    "profile_info",
+    "marks",
+    "collection_of_coins",
+]
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+for name_of_var in variable_names:
+    var_value = globals()[name_of_var]
+
+    if isinstance(var_value, (list, dict, set)):
+        sorted_variables["mutable"].append(var_value)
+    elif isinstance(var_value, (int, float, str, bool, tuple)):
+        sorted_variables["immutable"].append(var_value)
```

Comment:
There is no need to include the population logic, just fill `sorted_variables` with required variables as mentioned in the task description

Reply:
fill it in manually?